### DISCUSSION TITLE
Can we contribute to the wiki?

### DISCUSSION THREAD

**Discussion by bound-variable** [l4l/yofi#120] [Q&A] [answered]
I've contributed to the documentation of several projects, but I've never learned how to submit PRs for GitHub wikis.

Does this project allow for that?

I've made several emendations to the attached file.
[Configuration.md](https://github.com/l4l/yofi/files/9691467/Configuration.md)

I also thought it might be useful to add a note that the user can run `yofi apps --list` to see the apps and their respective filenames (for the blacklist).

**Comment by l4l** [comment, marked as answer]
> I've contributed to the documentation of several projects, but I've never learned how to submit PRs for GitHub wikis.
> 
> Does this project allow for that?

Ugh, don't know about that. I removed restriction for editing so feel free to update/add pages.

> I also thought it might be useful to add a note that the user can run yofi apps --list to see the apps and their respective filenames (for the blacklist).

Yeah, sure, though most of the args are undocumented in wiki as there's `--help` flag
